build.gradle.kts

```
@@ -64,16 +67,17 @@ dependencies {
     testCompile("com.atlassian.performance.tools:jira-software-actions:[1.0.0,2.0.0)")
     testCompile("org.hamcrest:hamcrest-library:1.3")
     testCompile("org.assertj:assertj-core:3.11.1")
-    testCompile("com.atlassian.performance.tools:ssh-ubuntu:0.2.0")
     testCompile("org.rnorth.duct-tape:duct-tape:1.0.7")
     testCompile("org.threeten:threeten-extra:1.5.0")
+    testCompile("com.github.docker-java:docker-java-core:[3.2.5, 4.0.0)")
+    testCompile("com.github.docker-java:docker-java-transport-zerodep:[3.2.5, 4.0.0)")
 }
 
 fun webdriver(): List<String> = listOf(
     "selenium-support",
     "selenium-chrome-driver"
 ).map { module ->
-    "org.seleniumhq.selenium:$module:3.11.0"
+    "org.seleniumhq.selenium:$module:[3.11.0, 3.999.999]"
```

If there is a reason why it's not 
```suggestion
    "org.seleniumhq.selenium:$module:[3.11.0, 4.0.0)"
```
then IMHO it should be noted in a comment

---

Good catch. They polluted their release channel with unstable versions like `4.0.0-alpha-1`

- [ ] put 3.999.999 rationale in the comment